haiku-apps/medo/medo-1.0.0~beta1.4.recipe

```diff
@@ -14,19 +11,31 @@ There are many bundled media effects, including:
 - Several special effects (blur, night vision, Chroma Key etc.)
 - Spatial tools to transform media (scale, rotate, position, crop)
 - Multiple text effects, including 3D extruded fonts
-- Audio effects (20 band equaliser, fade)
+- Audio effects (20 band equaliser, fade, IIR Filter)
 
 Medo can edit UHD 4K videos, export to any Haiku supported codec, is optimised \
 for many CPU-core systems, and has low system resource requirements.
 
-An SDK allows 3rd party developers to create addons and plugins."
+An SDK allows 3rd party developers to create addons and plugins.
+
+Changelog for Beta1-4.2
+- preview of media source
+- tag media sources
+- show clip trim/not visible markers
+- Srgb/Linear/Gamma plugin (video)
+- IIR Filter (audio)
+- Multi chroma key (video)
+- fixed multiple ffmpeg related crashes
+- fixed audio resampler issues when seeking
+- fixed zoom issues (use Z key for quick zoom)
+- miscellanous bug fixes"
 HOMEPAGE="https://github.com/smallstepforman/Medo"
 COPYRIGHT="2019-2021 Zen Yes Pty Ltd"
 LICENSE="MIT"
 REVISION="1"
-SOURCE_URI="https://github.com/smallstepforman/Medo/archive/R1.0.0-Beta1-3.tar.gz"
-CHECKSUM_SHA256="68c22862ec273b2d22a27be921fe770d7d3aa18f7a455d977ec0598c3104233e"
-SOURCE_DIR="Medo-R1.0.0-Beta1-3"
+SOURCE_URI="https://github.com/smallstepforman/Medo/releases/R1.0.0-Beta1-4.2.tar.gz"
```

Comment:
Build failed because it can't find the archive, could you revbump it with the correct link?
```diff
- SOURCE_URI="https://github.com/smallstepforman/Medo/releases/R1.0.0-Beta1-4.2.tar.gz"
+ SOURCE_URI="https://github.com/smallstepforman/Medo/archive/R1.0.0-Beta1-4.2.tar.gz"
```